kroxylicious/src/main/java/io/kroxylicious/proxy/service/ContributionManager.java

```diff
@@ -9,7 +9,6 @@
 import java.util.Map;
 import java.util.ServiceLoader;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.function.BiFunction;
 import java.util.function.Function;
 
 @SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
I realise I'm kicking up a lot of trouble already and possibly making myself unpopular, but can someone explain why this needs to be a singleton?

Reply:
No, I couldn't explain but decided not to kick that hornet nest when I came through last time.